core/src/main/kotlin/team/backend/context/RequestContext.kt

```diff
@@ -0,0 +1,41 @@
+package team.backend.context
+
+import kotlinx.coroutines.reactor.awaitSingle
+import org.springframework.web.server.ServerWebExchange
+import reactor.core.publisher.Mono
+import reactor.util.context.ContextView
+
+object RequestContext {
+
+    /**
+     * Get Value in Context
+     * @param key key of Context
+     * @return value in Context
+     */
+    suspend fun getValue(key: String): String {
+        return Mono
+            .deferContextual { ctx: ContextView ->
+                val authorization = ctx.get<String>(key)
```

Comment:
The function does not appear to be dependent on authorization.
The name seems to need to be changed.